python/lsst/dax/apdb/scripts/list_cassandra.py

```diff
@@ -23,17 +23,56 @@
 
 __all__ = ["list_cassandra"]
 
+from astropy.table import Table
+
 from ..cassandra import ApdbCassandra
 
 
-def list_cassandra(host: str) -> None:
+def list_cassandra(host: str, verbose: bool) -> None:
     """List APDB instances in Cassandra cluster.
 
     Parameters
     ----------
     host : `str`
         Name of one of the hosts in Cassandra cluster.
+    verbose : `bool`
+        If `True` provide detailed output.
     """
     databases = ApdbCassandra.list_databases(host=host)
-    for database in sorted(databases):
-        print(database)
+    if verbose:
+        columns = ["Keyspace", "Role", "Permissions"]
+        sort_columns = columns[:2]
+    else:
+        columns = ["Keyspace", "Roles[access]"]
+        sort_columns = columns[:1]
+    if databases:
+        table = Table(names=columns, dtype=[str] * len(columns))
+        for database in databases:
+            if database.permissions:
+                if verbose:
+                    for role, perm_list in database.permissions.items():
+                        permissions = ", ".join(sorted(perm_list))
+                        table.add_row([database.name, role, permissions])
+                else:
+                    roles = []
+                    for role, perm_list in database.permissions.items():
+                        access = _access(perm_list)
+                        roles.append(f"{role}[{access}]")
+                    table.add_row([database.name, " ".join(sorted(roles))])
```

Comment:
It might make more sense to have two different functions for verbose and non-verbose output instead of having multiple `if verbose` branches. They don't really have anything in common besides both being tables.

Reply:
For the "compressed" format, I think a comma-delimited list might be a bit easier to read, e.g.:
```
apdb                          apdb[update], super[manage]
```
instead of
```
apdb                          apdb[update] super[manage]
```